### ISSUE TITLE
Verification asked again even if all sessions are verified

### ISSUE THREAD

**Issue by tuxayo** [vector-im/element-web#14767]
### Description
After I logged in and verified my session with another device, it still asks to Verify this session.
On each app reload also.

### Steps to reproduce
- Open Element
> Verify this session
> Other users may not trust it
- click verify
> Confirm your identity by verifying this login from one of your other sessions, granting it access to encrypted messages.
- close
- see that in the list of my session, all are trusted

### Version information
- **Platform**: web at element.io

For the web app:

- **Browser**: Firefox 79.0b8
- **OS**: EndeavourOS (Arch Linux based)
- **URL**: app.element.io

**Comment by tuxayo** [comment]
logs sent

Additional info:

- trying to reverify triggers nothing on the other devices

- Settings => Security & Privacy => «Cross-signing and secret storage are not yet set up.»
```

Cross-signing public keys: | in memory
```
Cross-signing private keys: | not found
Self signing private key: | cached locally
User signing private key: | not found locally
Session backup key: | cached locally, well formed
Secret storage public key: | in account data
Homeserver feature support: | exists
```

Is this related?

- My previous riot.im session with which I did the verification also tells me to verify

**Comment by tuxayo** [comment]
I actually also have a similar issue with RiotX. In the last few months I did few tests of cross signing but never took the time to get the full picture of my issues.

https://github.com/vector-im/riotX-android/issues/1348

**Comment by tuxayo** [comment]
Also: I can access my two years old encrypted messages from the new session of element web

**Comment by jryans** [comment]
>     * Settings => Security & Privacy => «Cross-signing and secret storage are not yet set up.»
>       Is this related?
> 
> ```
> Cross-signing public keys: | in memory
> Cross-signing private keys: | not found
> Self signing private key: | cached locally
> User signing private key: | not found locally
> Session backup key: | cached locally, well formed
> Secret storage public key: | in account data
> Homeserver feature support: | exists
> ```

Yes, this seems related: the web device does not have you user-signing private key available locally, and it's also not stored in secret storage on the server.

Possible ways to resolve this include:

* Verifying web with some device that _does_ have that private key
* Enable Secure Backup / secret storage on a device that _does_ have that private key
* Resetting cross-signing on web (warning, resets your verification state and message backup key, please ensure you have message keys in memory on the same device)

I don't think there's a bug code wise: it seems like a slightly wedged state due to earlier application versions, so for now I'll close this, but please let us know what happens.

**Comment by freemo** [comment]
I am having this exact same problem. Every time I login it asks me to verify. I have not logged in on any other locations however. I am self-hosting element but it works fine otherwise. If i give it my security key every time it does verify. I have tried resetting all my keys, the security keys and cross keys.. problem persists.

**Comment by jthomaschewski** [comment]
I still experience this issue with up-to-date Element Web/Android/Desktop versions (web v1.10.1, android v1.4.0).
Maybe this issue should be reopened or is there a fix or a follow-up issue?

Current behavior:

Prerequisites: Multiple verified devices, cross-signing activated
Steps:
- Signin new device
- Complete verification process (verify with existing device => emoji match etc.)
- Open app on any of the other verified devices => Request "new device" appears, asking for verification

When cancelling the request and looking in the "Devices" section, all devices are verified ("green")

Sometimes the request even appears on the same device, which was used for verification (as reported in #19639)

**Comment by Taubin** [comment]
> I still experience this issue with up-to-date Element Web/Android/Desktop versions (web v1.10.1, android v1.4.0). @jryans could you reopen this issue or is there a fix or a follow-up issue somewhere?
> 
> Current behavior:
> 
> Prerequisites: Multiple verified devices, cross-signing activated Steps:
> 
>     * Signin new device
> 
>     * Complete verification process (verify with existing device => emoji match etc.)
> 
>     * Open app on any of the other verified devices => Request "new device" appears, asking for verification
> 
> 
> When cancelling the request and looking in the "Devices" section, all devices are verified ("green")
> 
> Sometimes the request even appears on the same device, which was used for verification (as reported in #19639)

I'm experiencing the same exact thing on a brand new macbook pro. The old macbook has verified, shown the same icons on both. We've tried the verification multiple times and it still shows every time the laptop opens the chat.
